pom.xml

```diff
@@ -2780,6 +2780,11 @@
                 <version>${testcontainers.accumulo.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>com.jayway.jsonpath</groupId>
+                <artifactId>json-path</artifactId>
+                <version>2.9.0</version>
+            </dependency>
```

Comment:
we already have this dependency declared in the pom, so we shouldn't need to redeclare it here.